main/ertp/guide/PursesAndPayments.md

```diff
@@ -0,0 +1,317 @@
+  - ```js
+    // Withdraw 3 Quatloos from a purse.
+    const newPayment = quatloosPurse.withdraw(quatloosAmountMath.make(3));
+    ```
+- [`purse.deposit(payment, optAmount)`](https://agoric.com/documentation/ertp/api/purse.html#purse-deposit-payment-optamount)
+  - Deposit the digital asset contents of the `payment` into this `purse`,
+    returning a description of the `payment` balance as an `amount`. If the optional argument
+    `optAmount` does not equal the `payment` balance,  or if `payment`
+    is an unresolved promise, it throws an error.
+  - ```js
+    const quatloosPurse = quatloosIssuer.makeEmptyPurse();
+    const quatloosPayment = quatloosMint.mintPayment(quatloosAmountMath.make(123));
+    const quatloos123 = quatloosmountMath.make(123);
+    // Deposit a payment for 123 quatloos into the purse. Ensure that this is the amount you expect.
+    quatloosPurse.deposit(quatloosPayment, quatloos123);
+    const secondPayment = quatloosMint.mintPayment(quatloosAmountMath.make(100));
+    // Throws error since secondPayment is 100 Quatloos and quatloos123 is 123 Quatloos
+    quatloosPurse.deposit(secondPayment, quatloos123);
+    ```
+- [`purse.getDepositFacet()`](https://agoric.com/documentation/ertp/api/purse.html#purse-makedepositfacet)
+  - Returns a deposit-only facet on the `purse` that can be given
+    to other parties. This lets them make a deposit to the `purse`, but not make
+    withdrawals from it or get its balance. Note that the command to add a `payment`'s
+    assets via a `DepositFacet` is not `deposit()` but `receive()` as shown here.
+  - ```js
+     const depositOnlyFacet = purse.getDepositFacet();
+     // Give depositOnlyFacet to someone else. They can pass a payment
+     // that will be deposited:
+     depositOnlyFacet.receive(payment);
+     ```
+
+In addition, the method to create a new, empty, `purse` is called on an `issuer`:
+- [`issuer.makeEmptyPurse()`](https://agoric.com/documentation/ertp/api/issuer.html#issuer-makeemptypurse)
+  - Returns a new empty `purse` for storing digital assets of the `brand` the `issuer` is associated with.
+  - ```js
+    // The new purse can only contain assets of the Quatloos brand.
+    const quatloosPurse = quatloosIssuer.makeEmptyPurse();
+    ```
+## Payments
+
+![Payment methods](./assets/payment.svg)   
+
+Payments hold digital assets intended to be transferred to another party.
+They are linear, meaning that either a `payment` has its full
+original balance, or it is used up entirely. It is impossible to
+partially use a `payment`. 
+
+In other words, if you create a `payment` containing
+10 Quatloos, the `payment` will always either contain 
+10 Quatloos or it will be deleted from its `issuer` records and no
+longer have any value. While a `payment` can be either combined with others or
+split into multiple `payments`, in both cases the original `payment(s)`
+is/are burned and the results put in one or more new `payments`.
```

Comment:
```suggestion
is/are deleted and the results put in one or more new `payments`.
```
Not burned. Let's reserve the word burned for when digital assets are destroyed.